### ISSUE TITLE
Documentation is confusing

### ISSUE THREAD

**Issue by azechini** [microsoft/WSL#8635]
I apologize if I am being a bit crass here, but I am thoroughly confused by the text "required optional components."

What does this mean? Are the components required? Are they optional? What are the components?

This text is found in the "Install WSL Command" section, right after the "wsl --install" code snippet. Again, I have no idea what this means, but I assume it may refer to the "Manual installation steps for older versions of WSL". Specifically, steps 1 and 3 that enable WSL and the Virtual Machine Platform, respectively. However, on that page it clearly states that steps 1 and 3 **ARE** required, using the language, "you must," which is quite clear.

If my assumption is correct, then it begs another question: why aren't these required steps on this top-level page??? 

Again, I apologize for being somewhat crass, but myself and a few others with whom I work have been struggling to install WSL on a Windows 10 laptop (that meets the prerequisites, according to this page) and I believe this is the source of our frustration. My coworker has hit the BSOD 3 times while trying to install WSL. He had to get the IT department involved, and it was only after the third BSOD they realized they needed to enable a few options for WSL to work properly.

I would greatly appreciate any clarity that can be provided.

**Comment by Biswa96** [comment]
> I am thoroughly confused by the text "required optional components."

Run optionalfeatures.exe, it will show the optional features. This may explain in details what optional features are https://www.howtogeek.com/250228/what-windows-10s-optional-features-do-and-how-to-turn-them-on-or-off/.

tl;dr The `wsl --install` command enables those "required optional components" automatically so that users don't have to enable them manually.

**Comment by NotTheDr01ds** [comment]
First, just a heads-up that there's a [separate Github repo for Doc issues](https://github.com/MicrosoftDocs/WSL/issues).

I'm assuming that you are referring to the oxymoron "required optional", right?

> `wsl --install`
> This command will enable the required optional component

Yes, it's confusing, and an oxymoron.  Put differently, they are "optional" features in Windows, but *required* for WSL.  It might be more clear if it was worded:

> This command will enable the Windows Features which are necessary for WSL.

And yes, as you mention, this means the features:

* Windows Subsystem for Linux
* Virtual Machine Platform

> [W]hy aren't these required steps on this top-level page?

Because `wsl --install` does it for you, there's really no need to cover it separately unless you are doing the "Manual Installation."

> I believe this is the source of our frustration. My coworker has hit the BSOD 3 times while trying to install WSL.

It seems unlikely, since `wsl --install` (as an Administrator, at least) should enable these for you, *assuming* that the user has Administrative access.  If not, then yes, you would need to drop back (as it sounds like the IT folks did) to the Manual Installation, and enable those features separately.

**Comment by benhillis** [comment]
Thanks for the feedback. @craigloewen-msft - we should update this, or if somebody in this thread would like to take a stab, we do accept PR's.

**Comment by craigloewen-msft** [comment]
Created an issue on the Docs repo for this: https://github.com/MicrosoftDocs/WSL/issues/1576 and a PR to address this, welcome to have any comments. 

https://github.com/MicrosoftDocs/WSL/pull/1577

**Comment by NotTheDr01ds** [comment]
@craigloewen-msft  Thanks Craig - Beat me to it!
